### PR TITLE
Feat #1813 : Add optional column for players uuids

### PR DESCRIPTION
**Pull request by Xephi**
Look at #1813

### CODE REVIEW

File: src/main/java/fr/xephi/authme/util/Utils.java [hunk 1]
```diff
@@ -106,4 +107,15 @@ public static boolean isCollectionEmpty(Collection<?> coll) {
...
         return StringUtils.isEmpty(email) || "your@email.com".equalsIgnoreCase(email);
     }
+
+    /**
+     * Returns whether the given string as an UUID or null
+     *
+     * @param string the uuid to parse
+     * @return parsed UUID if succeed or null
+     */
+    public static UUID parseUuidSafely(String string) {
```
Comment (sgdc3) [inline src/main/java/fr/xephi/authme/util/Utils.java]: Use UuidUtils directly

File: src/main/java/fr/xephi/authme/settings/properties/DatabaseSettings.java [hunk 2]
```diff
@@ -129,6 +129,10 @@
     public static final Property<String> MYSQL_COL_LASTLOC_PITCH =
         newProperty("DataSource.mySQLlastlocPitch", "pitch");
 
+    @Comment("Column for storing players uuids")
```
Comment (sgdc3) [inline src/main/java/fr/xephi/authme/settings/properties/DatabaseSettings.java]: Make clear that this column is optional

File: src/main/java/fr/xephi/authme/util/Utils.java (hunk 1 above)
Comment (Xephi) [reply src/main/java/fr/xephi/authme/util/Utils.java]: I'd like to keep a separation between Optional method and nullable method, maybe you want me to move this method to UuidUtils instead?

File: src/main/java/fr/xephi/authme/settings/properties/DatabaseSettings.java (hunk 2 above)
Comment (Xephi) [reply src/main/java/fr/xephi/authme/settings/properties/DatabaseSettings.java]: Nice catch

File: src/main/java/fr/xephi/authme/util/Utils.java (hunk 1 above)
Comment (sgdc3) [reply src/main/java/fr/xephi/authme/util/Utils.java]: Yeah, that sounds better :)

**Comment by sgdc3** [comment]
@Xephi bump?

**Comment by Xephi** [comment]
Sorry i completly forgot about that :p can you tell me why my todo comment is not ok for CodeClimate? :D

**Comment by ljacqu** [comment]
Please just resolve the todo. It‘s a small thing to fix. Personally I would drop the Optional return type

**Comment by sgdc3** [comment]
@Xephi bump

**Comment by Xephi** [comment]
Soz, completly forgot that one, done

**Pull request merged**